src/main/java/blackjack/domain/Result.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain;
+
+import java.util.function.BiPredicate;
+import java.util.stream.Stream;
+
+public enum Result {
+    WIN("승", Result::isWin),
+    LOSE("패", Result::isLose),
+    DRAW("무", Result::isDraw),
+    ;
+    private final String message;
+    private final BiPredicate<Integer, Integer> predicate;
+
+    Result(String message, BiPredicate<Integer, Integer> predicate) {
+        this.message = message;
+        this.predicate = predicate;
+    }
+
+    public static Result getInstance(int playerPoint, int dealerPoint) {
+        return Stream.of(Result.values())
+            .filter(it -> it.predicate.test(playerPoint, dealerPoint))
+            .findAny()
+            .orElse(null);
```

Comment:
코멘트 달면서 null을 리턴하도록 했는데 오류를 던져도 됩니다 :) 
```suggestion
            .orElseThrow(IllegalArgumentException::new);
```

Reply:
코드의 성질 상 예외를 던지는 게 더 나아 보이네요